### PR TITLE
Blog: Camel Quarkus 1.6.0

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+We are pleased to announce the release 1.6.0 of Camel Quarkus! Here are the highlights.
+
+## Java 11 required
+
+Having deprecated Java 8 [long time ago](https://camel.apache.org/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/) and
+following the same requirement by Quarkus, Camel Quarkus now requires Java 11+.
+
+## Support for more Camel components
+
+Camel Quarkus brings support for three new Camel bits:
```
Comment (jamesnetherton) [inline content/blog/2021/01/camel-quarkus-release-1.6.0/index.md:21]:
```suggestion
Camel Quarkus brings support for three new Camel components:
```

**Review by jamesnetherton — APPROVED** [review]
I wonder if we should add a little note about camel-fop as per [this thread](https://github.com/apache/camel-quarkus/issues/2113#issuecomment-756807434)?

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md (hunk 1 above)
Comment (ppalaga) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md:21]: Some time ago, I started using "bits" when I speak about languages, data formats and "others" in addition to components. Does "bits" sound too strange? I do not mind changing it here, but would still be nice to hear your your native speaker's opinion about "bits" :)

**Comment by ppalaga** [comment]
> I wonder if we should add a little note about camel-fop as per [this thread](https://github.com/apache/camel-quarkus/issues/2113#issuecomment-756807434)?

+1, let me add a sentence.

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md (hunk 1 above)
Comment (ppalaga) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md]: Replaced in 857ef0c

**Comment by ppalaga** [comment]
> > I wonder if we should add a little note about camel-fop as per [this thread](https://github.com/apache/camel-quarkus/issues/2113#issuecomment-756807434)?
> 
> +1, let me add a sentence.

Added a Known issue section in 857ef0c

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md (hunk 1 above)
Comment (jamesnetherton) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md]: 'bits' is like a casual, catch-all expression that can mean pretty much anything you like. So it doesn't really describe what three new things we have added support for.

As ever, I'm pretty relaxed about the language. So it's your choice whether to change it or not.

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+* [JSONATA](https://camel.apache.org/camel-quarkus/latest/reference/extensions/jsonata.html)
+* [Vert.x Kafka](https://camel.apache.org/camel-quarkus/latest/reference/extensions/vertx-kafka.html)
+* [Redis](https://camel.apache.org/camel-quarkus/latest/reference/extensions/redis.html) (JVM only)
+
+Components newly supported in native mode:
+
+* [Azure Storage Blob Service](https://camel.apache.org/camel-quarkus/latest/reference/extensions/azure-storage-blob.html)
+* [Debezium MongoDB Connector](https://camel.apache.org/camel-quarkus/latest/reference/extensions/debezium-mongodb.html)
+* [Google BigQuery](https://camel.apache.org/camel-quarkus/latest/reference/extensions/google-bigquery.html)
```
Comment (aldettinger) [inline content/blog/2021/01/camel-quarkus-release-1.6.0/index.md:31]: The google big query page indicates nativeSince = 1.5.0. Maybe we need to remove from release note ?

**Review by aldettinger — APPROVED** [review]
Just a question, otherwise looks good to me :+1:

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+summary: "The highlights of Camel Quarkus 1.6.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+We are pleased to announce the release 1.6.0 of Camel Quarkus! Here are the highlights.
+
+## Java 11 required
+
+Having deprecated Java 8 [long time ago](https://camel.apache.org/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/) and
```
Comment (aldettinger) [inline content/blog/2021/01/camel-quarkus-release-1.6.0/index.md:16]: Any reason to link on the camel website directly ?
I'm curious whether "/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/" would also work ?

Comment (ppalaga) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md:16]: Let me try

Comment (aldettinger) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md]: yes it does :+1:

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md (hunk 2 above)
Comment (aldettinger) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md]: Ah ok, the jvmSince was not correct. It is fixed in this [commit](https://github.com/apache/camel-quarkus/commit/7d8d55f44ccb79fe688d3d2e13e818c6756035cf).

**Review by zbendhiba — APPROVED** [review]

File: content/blog/2021/01/camel-quarkus-release-1.6.0/index.md (hunk 2 above)
Comment (ppalaga) [reply content/blog/2021/01/camel-quarkus-release-1.6.0/index.md]: Yes, exactly.

**Comment by ppalaga** [comment]
Thanks for the suggestions, everybody!

**Pull request merged**